api/DHSC.FingertipsNext.Modules.DataManagement/Controllers/V1/DataManagementBatchController.cs

```diff
@@ -22,4 +26,34 @@ public async Task<IActionResult> ListBatches()
 
         return new OkObjectResult(await dataManagementService.ListBatches(indicatorIds));
     }
+
+    [HttpDelete]
+    [Route("{batchId}")]
+    [ProducesResponseType(typeof(void), StatusCodes.Status200OK)]
+    public async Task<IActionResult> DeleteBatch([FromRoute] string batchId)
+    {
+        if (string.IsNullOrEmpty(batchId))
+            return new BadRequestObjectResult(new SimpleError
+            {
+                Message = "batchId is required"
+            });
+
+        //TODO: Add user ID
```

Comment:
Can we make this a comment that references which ticket this will be added under, rather than a generic "TODO" please?
```suggestion
        // User ID will be added under DHSCFT-930
```